schemas/context/namespace.schema.json

```diff
@@ -36,5 +36,5 @@
       "$ref": "#/definitions/namespace"
     }
   ],
-  "meta:status": "experimental"
+  "meta:status": "stabilizing"
```

Comment:
As above, this is pending on #203 👎

Reply:
Same comment as for datasource, this is just documentation we should not hold up our milestone.

Reply:
No preference